Automattic/wp-calypso
# Site Settings: Remove Jetpack Scan from this world

This PR disables Jetpack Scan in all environments. Previously it was enabled for our testing and development environments, but not for the production one. 

Fixes #11617.

To test:
* Checkout this branch
* Go to `/settings/security/$site`, where `$site` is one of your Jetpack sites.
* Verify the `Jetpack Scan` card is no longer there.
